presto/presto.go

```diff
@@ -103,13 +103,17 @@ const (
 	prestoCatalogHeader     = "X-Presto-Catalog"
 	prestoSchemaHeader      = "X-Presto-Schema"
 	prestoSessionHeader     = "X-Presto-Session"
+	prestoSetCatalogHeader  = "X-Presto-Set-Catalog"
+	prestoSetSchemaHeader   = "X-Presto-Set-Schema"
 
+	// KerberosEnabledConfig is a param name
```

Comment:
group both params in single code section and just add single comment to name a section `// Param names`

Reply:
All exported vars and functions need to have a comment that begins with their name and reads like a proper english sentence. These are an example where the name is pretty much self explanatory, so the comment doesn't add much. We have three options here:
* change the comment to one that disables the linter rule on this line
* disable this specific linter for the whole repo
* leave these comments as is

WDYT?

Reply:
I would go for `disable this specific linter for the whole repo`. This one specifically makes more mess than benefit. Instead IMO we should use review-process to ensure that comments are present where they actually improve code readability (whether it is exported function, private one, or some code snippet within a function)

Reply:
I reverted these changes. It turned out that by default `golangci-lint` ignores these specific errors from the `golint` linter but my editor overrides this. The full list is here: https://golangci-lint.run/usage/configuration/

Since we don't have any linters in the pipeline I'm not going to add any configs here. This is out of scope of this PR.